backstop_data/engine_scripts/puppet/search/results-more-actions-add-contributions-modal.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+module.exports = async (engine, scenario, vp) => {
+  await require('./results-more-actions-dropdown')(engine, scenario, vp);
+  await engine.click('span.crm-hover-button a[title="Add Contribution"]');
+  await engine.waitFor('.CRM_Contribute_Form_Contribution', { visible: true });
+  await require('../common/open-accordions')(engine, scenario, vp);
+  await engine.waitFor('[alt="loading"]', { hidden: true });  
```

Comment:
what's this selector for?

Reply:
For loading screen to disappear!

Reply:
<img width="1413" alt="alt loading - add contributions" src="https://user-images.githubusercontent.com/3340537/41546789-314395a2-733c-11e8-92a3-d4713b8e06c7.png">